internal/bucket/bucket.go

```diff
@@ -24,9 +24,9 @@ var (
 	CID -> reference to content (map[string]interface{})
 */
 type BucketContent struct {
```

Comment:
If this is added as a protobuf type within `proto/common` we would have easier Json serialization, and extensibility with the definition across other modules within motor.

Reply:
I think youre right that this should be moved to a proto def, I think the same can be said about some of the `Object` structs as well.